moped-editor/src/views/projects/projectView/ProjectFundingTable.js

```diff
@@ -41,19 +35,16 @@ import {
   UPDATE_PROJECT_FUNDING,
   ADD_PROJECT_FUNDING,
   DELETE_PROJECT_FUNDING,
-  UPDATE_FUNDING_TASK_ORDERS,
 } from "../../../queries/funding";
 
 import { getDatabaseId, useUser } from "../../../auth/user";
 import ProjectSummaryProjectECapris from "./ProjectSummary/ProjectSummaryProjectECapris";
-import TaskOrderAutocomplete from "../signalProjectTable/TaskOrderAutocomplete";
```

Comment:
task order picking has moved to work activity records. am going to open a separate issue to migrate the task order data and fix up the project list view.